documentation/asciidoc/topics/ref_user_roles_permissions.adoc

```diff
@@ -0,0 +1,41 @@
+[id='user-roles-permissions_{context}']
+= User roles and permissions
+
+[role="_abstract"]
+{ispn_operator} uses the `ClusterRoleMapper` mechanism to associate roles with permissions, which makes a default set of roles available.
```

Comment:
For client-cert we will actually use the `common-name-role-mapper`. So maybe it's best to just say something like the following roles are available by default ...